app/controllers/users.py

```diff
@@ -171,11 +171,12 @@ def post(self):
         if not user:
             return dict(status='fail', message="login failed"), 401
 
-        if not user.verified:
```

Comment:
I believe we still need to check if a user is verified. (I have not worked with the codebase for a while, I could be wrong)

Reply:
Yes it is true. I have modified it . Thank you on that.